products/bbc.toml

```
@@ -0,0 +1,75 @@
+
+[rubric.data-breaches]
+value = "no"
+citations = [""]
+
+[rubric.data-collection-reasoning]
+value = "mostly"
+citations = ["We have to have a valid reason to use your personal information. It's called the \'lawful basis for processing\'. Sometimes we might ask your permission to do things, like when you subscribe to an email. Other times, when you\'d reasonably expect us to use your personal information, we don't ask your permission, but only when the law says it\'s fine to use it, and it fits with the rights you have."]
+notes = ["The full list of why the BBC collects personal information is in Section 6 of the Privacy Policy."]
+
+[rubric.data-deletion]
+value = "yes-automated"
+citations = ["This depends on what information you're talking about. You can delete your account. Your account information is immediately deleted. Keep in mind we keep a record of how you've used our services, but this information can't be linked back to you (and) we also keep anything you\'ve uploaded or commented on. We might have collected other personal information that you might ask to be deleted that has nothing to do with BBC account."]
+
+[rubric.history]
+value = "last-modified"
+citations = [""]
+
+[rubric.law-enforcement]
+value = "not-specified"
+citations = ["Sometimes by law we have to pass on your information to other organisations. We might also share your information if we have to by law, or when we need to protect you or other people from harm."]
+
+[rubric.list-collected]
+value = "generally"
+citations = ["We\'ll give you details about why we need your personal information and how we'll use it before you begin, unless it\'s obvious.
+a. Information that you give us
+We might ask for your name and contact details, your date of birth or financial details, depending on what you're doing. If you apply to take part in a programme on politics, for example, we might ask you about your political opinions. Information you give us might include information in a creation you share with us, such as people's names that appear in the creation.
+When you register for a BBC account, or update your details or settings we ask for some personal information, like your email address and age.
+b. Device information
+Devices are lots of things like your computer, your mobile, your TV, your tablet and your voice-enabled device.
+We automatically collect some technical information from these devices and web browsers even when you\'re not signed into to a BBC account. This might include IP (internet protocol) address, device ID, app ID, vendor ID, advertising ID
+c. Information on your activities outside the BBC when you talk about us
+Like if you've mentioned us in a Twitter post, we\'ll collect your Twitter handle.
+d. Information on how you use our services
+Like the articles you read and the programmes you watch."]
```

(as follows from the previous suggestion. FYI, I haven't tested it, but it _should_ work. See this helpful cheatsheet: https://toml.io/en/)
```suggestion
Like the articles you read and the programmes you watch."""]
```